OpenRA.Game/Actor.cs

```diff
@@ -284,25 +284,30 @@ public void ChangeOwner(Player newOwner)
 		{
 			World.AddFrameEndTask(w =>
```

Comment:
Style nit: omit the braces for single line statements: `World.AddFrameEndTask(w => ChangeOwnerSync(w, newOwner));`